docs/APIs/tasks/Timeout.md

```diff
@@ -1,8 +1,8 @@
-# TimeOut
+# Timeout
 
 Use the Timeout interface to set up an interrupt to call a function after a specified delay.
 
-Any number of Timeout objects can be created, allowing multiple outstanding interrupts at the same time.
+You can create any number of Timeout objects, allowing multiple outstanding interrupts at the same time.
```

Comment:
After much discussion in the office, the engineers say our docs should say "Timeout".

I don't see any yellow.